pkgs/applications/audio/picard/default.nix

```diff
@@ -13,7 +22,17 @@ in pythonPackages.buildPythonApplication rec {
     sha256 = "0xalg4dvaqb396h4s6gzxnplgv1lcvsczmmrlhyrj0kfj10amhsj";
   };
 
-  nativeBuildInputs = [ gettext qt5.wrapQtAppsHook qt5.qtbase ];
+  nativeBuildInputs = [ gettext qt5.wrapQtAppsHook qt5.qtbase ]
+    ++ stdenv.lib.optionals (pyqt5.multimediaEnabled) [
+      qt5.qtmultimedia.bin
+      wrapGAppsHook
```

Comment:
```suggestion
```

Reply:
Thanks for catching that @jonringer , I had this left over from tests I made with other environmental variables. Only this one is unneeded and without double wrapping doesn't occur - fixed in last push.